crates/sui-protocol-config/src/lib.rs

```diff
@@ -546,10 +556,10 @@ pub struct ProtocolConfig {
     hmac_hmac_sha3_256_input_cost_per_block: Option<u64>,
 }
 
-const CONSTANT_ERR_MSG: &str = "protocol constant not present in current protocol version";
-
 // feature flags
 impl ProtocolConfig {
+    const CONSTANT_ERR_MSG: &str = "protocol constant not present in current protocol version";
```

Comment:
nit: for macro hygiene reasons,  I would put this constant definition inside the `impl` block that you create in the macro.